### PR TITLE
update README

### PR DESCRIPTION
**Pull request by strausr**
### Brief Summary of Changes
update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,258 +1,103 @@
...
 
-For Node.js, Cloudinary provides an extension for simplifying the integration even further.
 
-## Getting started guide
-![](https://res.cloudinary.com/cloudinary/image/upload/see_more_bullet.png)  **Take a look at our [Getting started guide for Node.js](https://cloudinary.com/documentation/node_integration#node_js_getting_started_guide)**.
 
+## Version Support
+| SDK Version   | node 6-14 |
+|---------------|-----------|
+| 1.28.1 & up    | X        |
```
Comment (maoznir) [inline README.md:35]: 1. Show be a V instead of an X
2. What about Node 16?

**Review by maoznir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (strausr) [reply README.md]: updated. Are tests only run on 6-14 so officially this is what we support

File: README.md [hunk 2]
```diff
@@ -1,258 +1,103 @@
...
 
-For Node.js, Cloudinary provides an extension for simplifying the integration even further.
 
-## Getting started guide
-![](https://res.cloudinary.com/cloudinary/image/upload/see_more_bullet.png)  **Take a look at our [Getting started guide for Node.js](https://cloudinary.com/documentation/node_integration#node_js_getting_started_guide)**.
 
+## Version Support
+| SDK Version   | node 6-14 |
+|---------------|-----------|
+| 1.28.1 & up   | V         |
```
Comment (maoznir) [inline README.md:35]: What happens at versions < 1.28.1 ?

**Review by patrick-tolosa — CHANGES_REQUESTED** [review]
Please adjust the versions

**Pull request merged**